index.html

```diff
@@ -3,6 +3,10 @@
     <meta http-equiv="Content-Type" content="text/html;charset=UTF-8">
     <title>JS fun!</title>
     <script type="text/javascript" src="main.js"></script>
+	<style> 
+	h1
+	{ color: blue}
+	</style>
```

Comment:
Formatting: particular items should be indented from the parent tag. 

```
<style>
  h1 {
     key: value; 
  }
</style>
```

Reply:
Ok, fixed